build/signedbuild.yml

```diff
@@ -319,6 +320,22 @@ jobs:
       PathtoPublish: 'src\CLI_Installer\bin\Release\AxeWindowsCLI.zip'
       ArtifactName: 'CLI-zip'
 
+  - task: CopyFiles@2
+    displayName: 'Copy CLI_Full Files for Signing Validation'
+    inputs:
+      SourceFolder: '$(Build.SourcesDirectory)\src'
+      Contents: |
+       CLI_Full\bin\release\netcoreapp3.1\win7-x86\?(*.exe|*.dll)
+       !CLI_Full\bin\release\netcoreapp3.1\win7-x86\?(CommandLine.dll|Newtonsoft.Json.dll)
```

Comment:
CommandLine and Newtonsoft.Json.dll won't be signed here because we don't ship them.